core/domain/exp_jobs_one_off_test.py

```diff
@@ -2695,3 +2695,126 @@ def test_no_action_is_performed_for_deleted_exploration(self):
         run_job_for_deleted_exp(
             self,
             exp_jobs_one_off.InteractionCustomizationArgsValidationJob)
+
+
+class EmptySubtitledHtmlAuditJobTest(test_utils.GenericTestBase):
+    """Tests for the EmptySubtitledHtmlAuditJob one off job."""
+    def setUp(self):
+        super(EmptySubtitledHtmlAuditJobTest, self).setUp()
+        self.EXP_ID_1 = 'exp1'
+        self.signup(self.OWNER_EMAIL, self.OWNER_USERNAME)
+        self.owner_id = self.get_user_id_from_email(self.OWNER_EMAIL)
+        self.process_and_flush_pending_tasks()
+
+    def test_audit_job_with_empty_html_exploration_returns_content_id(self):
+
```

Comment:
Add a docstring.

Reply:
I think test functions don't need docstring.

Reply:
All other test functions in the file have a docstring, so just for consistency, it would be better to have one here too!

Reply:
Sure, I'll add one! :)

Reply:
Done!